tests/tests_integration/test_api/test_annotations.py

```diff
@@ -321,13 +324,7 @@ def test_retrieve_multiple(self, cognite_client: CogniteClient, base_annotation:
         ids = [c.id for c in created_annotations]
         retrieved_annotations = cognite_client.annotations.retrieve_multiple(ids)
         assert isinstance(retrieved_annotations, AnnotationList)
-
-        # TODO assert the order and do without sorting
-        # as soon as the API is fixed
```

Comment:
Nice fix.

Reply:
This was unrelated actually, I just checked if the api response had been fixed 😄